apps/web/src/libs/event-emitter.ts

```diff
@@ -0,0 +1,35 @@
+export interface Subscriber {
+  (...args: any[]): void
+}
+
+export class EventEmitter {
+  private _subscribers: { [event: string]: Subscriber[] } = {}
+
+  public subscribe(event: string, function_: Subscriber) {
```

Comment:
`function_` is a little ambiguous, unless it means something I don't understand, wouldn't this just be something like `callback`?

Reply:
Good point! I'll rename it to `subscriber`.